charts/shoot-cloud-config/charts/original/templates/docker/_docker.service

```diff
@@ -1,9 +1,3 @@
 {{- define "docker" -}}
```

Comment:
Can't we delete the whole file?

Reply:
Not sure, no strong opinion about this one as it does not change the behaviour anyways.